### PR TITLE
Tidying up LD2420 doc

### PR DESCRIPTION
**Pull request by kbx81**
## Description:

Fixing a few details I missed during review.

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Review by nagyrobi — DISMISSED** [review]

**Comment by nagyrobi** [comment]
_Operating Mode_ is a bit confusing. How do you set it?

**Comment by kbx81** [comment]
> _Operating Mode_ is a bit confusing. How do you set it?

That's the `select` component which selects the operating mode of the sensor -- it's detailed in part of the document. However, that's not something this PR is addressing; it's just making the example consistent with our other docs and adding in a few links to component details.

**Comment by nagyrobi** [comment]
I guess what's missing from making it clear is the "All other options from _select_" link, which appears for all the other components, except this one.

**Comment by kbx81** [comment]
Oh my gosh -- I'm sorry. I misunderstood your first comment...and totally overlooked that one. Good catch! I've pushed an update...thank you! 😄

File: components/sensor/ld2420.rst [hunk 1]
```diff
@@ -189,6 +180,7 @@ Configuration variables:
 ************************
 
 - **operating_mode** (*Required*): Provisions the operating mode select component.
+- All other options from :ref:`Select <config-select>`.
```
Comment (jesserockz) [inline components/sensor/ld2420.rst]: These are part of the keys like `operating_mode`, not extra config items alongside it
```suggestion
  All other options from :ref:`Select <config-select>`.
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

File: components/sensor/ld2420.rst (hunk 1 above)
Comment (jesserockz) [reply components/sensor/ld2420.rst]: The same applies to all the rest too, each sub-number etc should have this text in each one.

Comment (kbx81) [reply components/sensor/ld2420.rst]: Wow, yeah, thank you...wasn't thinking about it in the right way. All fixed now, though. 😄

**Review by jesserockz — APPROVED** [review]

**Pull request merged**